tools/typescript2fabel.js

```diff
@@ -273,10 +286,28 @@ function visitModule(node) {
 
 function visitFile(node) {
     var modules = [];
-    ts.forEachChild(node, function(node) {
+	var emptyModule = getModule(node, true);
+    ts.forEachChild(node, function(node) {			    
         switch (node.kind) {
+            case ts.SyntaxKind.InterfaceDeclaration:
+				if (modules.length == 0 ) { modules.push( emptyModule ); }
+                emptyModule.interfaces.push(visitInterface(node));
+                break;
+            case ts.SyntaxKind.VariableStatement:
+				if (modules.length == 0 ) { modules.push( emptyModule ); }
+                getVariables(node).forEach(x =>
+                    emptyModule.properties.push(x));
+                break;
+            case ts.SyntaxKind.FunctionDeclaration:
+				if (modules.length == 0 ) { modules.push( emptyModule ); }
+                emptyModule.methods.push(getMethod(node, true));
+                break;
```

Comment:
Besides `Import` attribute, there'll be a `Global` attribute. Maybe we can use that for elements defined outside modules.